src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar bar--0" data-qa="big">
+      <div
+      alt="searchImage"
+      class="search-bar__icon-big"></div>
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/97547824/f0be21db-4c05-475d-88f8-c1ec28aafebe)

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/138720644/6e8bd67a-ab0f-4e03-b0cb-dc2cec2a763a)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/138720644/3ebcf1d2-d644-42fd-802d-1a518894e96a)

Reply:
I mean there is no reason to create an additional div element. background-image can be appied for input itself

Reply:
I understand your idea, but in the mock we need make in the input, we must make sure that the input cursor is near the placeholder, but if we give our input a background-image, then the result will be different ( I posted two screenshots for verification)


![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/138720644/82e3207b-4940-4fe7-af7f-a8b0463192a5)
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/138720644/9620d626-8d94-43fb-bb92-c93b3ca90882)